central/productusage/datastore/securedunits/datastore.go

```diff
@@ -0,0 +1,35 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// DataStore is the datastore for usage.
+//
+//go:generate mockgen-wrapper
+type DataStore interface {
+	// Persistent storage access:
+
+	// Get returns the channel, from which the metrics could be read.
+	Get(ctx context.Context, from *types.Timestamp, to *types.Timestamp) (<-chan *storage.SecuredUnits, error)
```

Comment:
When `Get` fails, we return a plain text error with 500 code. Otherwise, we start writing CSV to the output. Passing CSV writer closure to `Get` implies that the writer has been initialized, which means it started to write the output (BOM, CSV headers).
Making a lazy writer would require some cumbersome logic to only write the headers on the first record received.
Having a channel allows for separating the Get and Write, and simplifies the handler code.